code/clue.js

```diff
@@ -1,59 +1,238 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+// OBJECTS FOR ALL THE SUSPECTS:
 
 const mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
-  color: 'green',
+  color: '#82c59c',
   description: 'He has a lot of connections',
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  occupation: 'Entrepreneur',
+  weapon: 'bat',
+  favouriteWeapon: 'trophy'
 }
-
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: '#ffeb7a',
+  description: 'He is a dignified, dapper and dangerous military man',
+  age: 54,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel',
+  weapon: 'candlestick',
+  favouriteWeapon: 'knife'
+}
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: '#7ed4e6',
+  description: 'A grande dame, an elderly, attractive woman who nearly always maintains her dignity',
+  age: 63,
+  image: 'assets/peacock.png',
+  occupation: 'Socialite',
+  weapon: 'pistol',
+  favouriteWeapon: 'candlestick'
+}
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: '#75709e',
+  description: 'A quick-witted, professor with a bow tie and glasses',
```

Comment:
I L😍VE how creative you are with the descriptions. It makes it so fun to shuffle and read between the cards.